simplegrid/world.py

```diff
@@ -82,6 +93,10 @@ def step(self):
         for creature in born:
             self.add_new_creature(creature)
 
+        for creature in self.creatures.values():
```

Comment:
Debug info?

Reply:
Ja, dit is nog niet klaar, toch? Het is alleen om je een tussen kijkje te geven en context van wat er verder veranderd moet. Wat nu duidelijk is, denk ik.